CI/appveyor.after_success.ps1

```diff
@@ -142,8 +142,10 @@ if ("$Env:APPVEYOR_REPO_TAG" -eq "false" -and -Not $Script:PublicTestBuild) {
   dblsqd login -e "https://api.dblsqd.com/v1/jsonrpc" -u "${Env:DBLSQD_USER}" -p "${Env:DBLSQD_PASS}"
 
   if ($Script:PublicTestBuild) {
+    Write-Output "=== Generating a changelog ==="
+    $Script:Changelog = lua "$Env:APPVEYOR_BUILD_FOLDER\CI\generate-ptb-changelog.lua" --osx win -- architecture x86
```

Comment:
Is that `--osx win -- architecture x86` formatted correctly - I'd have thought it should be: `--os win --architecture x86` for the (32-Bit) Windows case...

Reply:
Good catch! Thanks